osquery/tables/networking/posix/curl.cpp

```diff
@@ -28,22 +26,22 @@ const std::string kOsqueryUserAgent{"osquery"};
 
 Status processRequest(Row& r) {
   try {
-    client client_;
-    client::response response_;
-    client::request request_(r["url"]);
+    osquery::http::Client client_;
+    osquery::http::Response response_;
+    osquery::http::Request request_(r["url"]);
 
     // Change the user-agent for the request to be osquery
-    request_ << header("User-Agent", r["user_agent"]);
+    request_ << osquery::http::Request::Header("User-Agent", r["user_agent"]);
 
     // Measure the rtt using the system clock
     time_point<system_clock> start = std::chrono::system_clock::now();
     response_ = client_.get(request_);
     time_point<system_clock> end = std::chrono::system_clock::now();
 
-    r["response_code"] = INTEGER(static_cast<int>(status(response_)));
+    r["response_code"] = INTEGER(static_cast<int>(response_.status()));
     r["round_trip_time"] =
         BIGINT(duration_cast<microseconds>(end - start).count());
-    r["result"] = static_cast<std::string>(body(response_));
+    r["result"] = response_.body();
```

Comment:
Consider this instead:
```
r["result"] = std::move(response_.body); // older Beast version
```
Alternatively, if you are using the latest Beast version:
```
r["result"] = std::move(response_.body());
```
And remove the member function `Response::body`